about.html

```diff
@@ -0,0 +1,116 @@
+                <li><a class="dark" href="#">News</a></li>
+                <li class="red"><a class="red" href="#">CC Campaign</a></li>
+            </ul>
+            </div>
+        </div>
+    </header>
+    <article class="sec1-cont">
+    <header class="sec1">
+        <h2 class="mobile">"Welcome! Define Tomorrow"</h2>
+        <h1>College of Science, Engineering and Technology 2023</h1>
+    </header>
+    
+        <p class="text-ab">
+            The School of Engineering offers National Diploma and Degree courses in 
+            Engineering through open distance learning (ODL) to students who are 
+            interested in becoming professional technicians and technologists in a 
+            number of engineering disciplines.
+        </p>
+        <div class="info">
+            <p class="mail">Please contact us per Email for any further questions 
+            about the enrollment</p>
+            <a href=""><p class="email">study-info@unisa.ac.za</p></a>
+        </div>
+    </article>
+</section>
+<section>
+    <div class="cont-sec3">
+        <div class="sec3">
+            <h3>The leading CODeL university</h3>
+            <img src="./Image/title-line.png" alt="">
+        </div>
+        <p class="text4">This year's ceremony marks a significant 150th anniversary milestone (#Unisa150). 
+            <span class="desktop">Our vision describes our wish to serve every country on the African continent while transcending 
+            language and cultural barriers. We are 100% committed to the communities and individuals we serve.</span></p>
+    </div>
+    <div class="logo-cont">
+        <div class="logo1">
+            <img src="./Image/uni-logo1.png">
+        </div>
+        <div class="wrap">
+                <div class="sec3-2">
+                    <h3>See the past years graduations.</h3>
+                    <img src="./Image/title-line.png" alt="">
+                </div>
+                <p class="text5">Take a look at our students celebrating and joining the alumni association.</p>
+                <div class="grad">
+                    <img src="./Image/graduation.jpg" alt="students graduation">
+                    <img src="./Image/graduation2.jpg" alt="students graduation">
+                </div>
+        </div>
+    </div>
+    <article class="partners1" id="sponsor">
+        <div class="sponsors">
+            <h3>Partners</h3>
+            <img src="./Image/title-line.png" alt="">
+        </div>   
+        <ul>
+            <li>ALUMNI</li>
+            <li>Funza Lushaka</li>
+            <li>Department of Arts and Culture</li>
+            <li>NSFAS</li>
+        </ul>
+    </article>
+    <div class="mobile footer">
+        <p class="f-item">2023 Creative Commons South Africa.<br> Some Rights Reserved.</p>
+        <div>
+            <img class="f-item cck" src="./Image/cck.png" alt="">SOUTH AFRICA
+        </div>
+        
+    </div>
+</section>
+<script src="about.js"></script>  
+</body>
```

Comment:
- [ ] I also noticed that on the `About page,` you're missing the `footer` section. So kindly include that in your design as well and remove the `Partners Section` so your design looks similar to the original design :100: 
- [ ] I also noticed that you have a horizontal scroll effect in your design so kindly fix this as well :100: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/235146159-16853f11-81f7-48da-b7a8-d10129619b0b.png)

- The Behance design
![image](https://user-images.githubusercontent.com/102699958/235146406-a7ec3c45-b420-4675-8c72-d1b6a7861f3d.png)